content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -201,6 +201,63 @@ them according to its `updateStrategy`.
 
 You can [perform a rolling update](/docs/tasks/manage-daemon/update-daemon-set/) on a DaemonSet.
 
+While working with the rolling updates of the DaemonSets, you will come across two important parameters: `maxSurge` and `maxUnavailable`.
+
+### maxSurge
+
+If you set this parameter to a number above 0, you allow the DaemonSet controller
+to create temporary duplicate Pods during a rollout.
+Because a DaemonSet aims to have one healthy Pod per node, this field also implies
+the number of _nodes_ that can have a duplicate Pod on them at any one moment
+during the rollout.
+While performing a rolling update, Kubernetes will keep creating new pods to replace the old ones. 
+`maxSurge` helps you to regulate the surge of additional pods beyond the desired count. 
+
+#### maxSurge and Pod failures
+
```

Comment:
I don't know if is useful to specify this for both `maxSurge` and `maxUnavailable`. This behaviour will occurr in all circumstances as both of the constraints cannot be 0. I think it is best to remove it or move this to general section just called `Pod failures`.